package/zshelf/package

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+# Copyright (c) 2020 The Toltec Contributors
+# SPDX-License-Identifier: MIT
+
+pkgnames=(zshelf)
+pkgdesc="Z-Library browser and downloader"
+url=https://github.com/khanhas/zshelf
+pkgver=0.1.0
+section=utils
+timestamp=2021-01-30T15:48Z
+maintainer="khanhas <xuankhanh963@gmail.com>"
+license=GPL-3.0
+depends=(node)
+
+source=(
+    https://github.com/khanhas/zshelf/archive/v0.1.0.zip
+    zshelf.draft
+    zshelf.png
+)
+sha256sums=(
+    95252da324dbaec419f277532afe29c75cbac901a7a5da828ea6abb620687611
+    SKIP
+    SKIP
+)
+
+image=qt:v1.1
+build() {
+    qmake zshelf.pro
+    make
+
+    # Get needed build packages
+    export DEBIAN_FRONTEND=noninteractive
+    apt-get update -y
+    apt-get install -y \
+        --no-install-recommends \
+        -o Dpkg::Options::="--force-confdef" \
+        nodejs npm
+
+    cd backend
+    npm install
+}
+
+package() {
+    install -d "$pkgdir"/opt/share/zshelf
+
+    cp -r "$srcdir"/backend "$pkgdir"/opt/share/zshelf/backend
+    cp "$srcdir"/config.json "$pkgdir"/opt/share/zshelf
+    install -D -m 755 -t "$pkgdir"/opt/share/zshelf "$srcdir"/zshelf
+
+    ln -s "$pkgdir"/opt/share/zshelf/zshelf "$pkgdir"/opt/bin
```

Comment:
You're including a buildtime specific path here. I suggest to remove the first `"$pkgdir"` as this will by taken over 1-to-1 to the device.

Reply:
Thank you for your review!
Which `"$pkgdir"` is that I need to remove and why only the first one? Did you mean remove all the prefixed `"$pkgdir"`?

Reply:
Oh I got it now. At the `ln` command isn't it? Github showed 3 lines above it and I was confused haha.

Reply:
Yes. Sorry that I didn't just point that out, was in a hurry and didn't want to make a wrong/untested suggestion.